include/email.js

```diff
@@ -61,7 +65,7 @@ module.exports = function EmailServiceModule(pb) {
      */
     EmailService.prototype.sendFromTemplate = function(options, cb){
         var self = this;
-        var ts   = new pb.TemplateService();
+        var ts   = new pb.TemplateService({ site: this.site });
```

Comment:
For us.  I won't pawn refactoring our poor code off onto you guys.  

However, would you do me a small favor and add the TODO in there with my comments?